Dockerfile

```diff
@@ -3,6 +3,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends openssh-client
 RUN go get github.com/mitchellh/gox
 RUN go get github.com/aktau/github-release
 RUN go get github.com/tools/godep
+RUN go get code.google.com/p/go.tools/cmd/cover
```

Comment:
Good callout @thaJeztah.  I will make a note to fix and maybe follow up with a PR to do so.